beacon/aeon_details.go

```diff
@@ -185,6 +197,45 @@ func loadAeonDetails(aeonDetailsFile *AeonDetailsFile, validators *types.Validat
 	return aeonDetails, nil
 }
 
+// Add aon aeon to the file, keeping a max of N aeons in the file (reading
+// from the file and appending if neccessary)
+func updateFileAeons(filePath string, max int, aeons ...*aeonDetails) {
```

Comment:
Function to ensure N (2) aeons are in the old aeons file

Reply:
Wasn't this change already merged?

Reply:
Ah, I realised that I didn't close that PR at the time I made the comment, yes it was merged previously and thus was not in this PR